recipe/meta.yaml

```
@@ -1,41 +1,41 @@
-{% set version = "0.17.1" %}
 {% set name = "meson-python" %}
+{% set version = "0.18.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/meson_python-{{ version }}.tar.gz
-  sha256: efb91f69f2e19eef7bc9a471ed2a4e730088cc6b39eacaf3e49fc4f930eb5f83
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
+  sha256: c56a99ec9df669a40662fe46960321af6e4b14106c14db228709c1628e23848d
 
 build:
-  skip: True # [py<37]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True # [py<38]
 
 requirements:
   host:
-    - meson >=0.63.3    # [py<312]
-    - meson >=1.2.3     # [py>=312]
     - pip
-    - pyproject-metadata >=0.7.1
     - python
+    - meson >=0.63.3    # [py<312]
+    - meson >=1.2.3     # [py>=312]
+    - packaging >=23.2
+    - pyproject-metadata >=0.9.0
     - setuptools
     - tomli >=1.0.0     # [py<311]
-    - packaging >=19.0
     # mesonpy takes care of creating the wheels without using
     # wheel. See
     # https://github.com/mesonbuild/meson-python/blob/main/mesonpy/_wheelfile.py
   run:
+    - python
     # meson >1.2 on py38 (on at least osx-arm64) will generate a SEGV
     # during tests.  Verified on 0.16.0.
     - meson 1.2         # [py<=38]
-    - meson >=0.63.3    # [py<312]
+    - meson >=0.63.3    # [py>38 or py<312]
```

```suggestion
    - meson >=0.64.0    # [py>38 or py<312]
```
https://github.com/mesonbuild/meson-python/blame/0.18.0/pyproject.toml#L38